source/macos/AdaptiveCards/AdaptiveCards/Renderers/TextInputRenderer.swift

```diff
@@ -160,9 +160,11 @@ class ACRTextInputView: NSTextField, InputHandlingViewProtocol {
     
     var maxLen: Int = 0
     var idString: String?
+    var config: RenderConfig?
     
```

Comment:
```suggestion
    private let renderConfig: RenderConfig?
    
```
This can be a private constant.